src/Components/itemDetail.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import '../App.css';
+import { useParams } from 'react-router-dom';
+import { getLocalToken } from '../lib/token.js';
+import { FirestoreDocument } from 'react-firestore';
+import { Link } from 'react-router-dom';
+
+function ItemDetailPage() {
+  let { docId } = useParams();
```

Comment:
I love that you're using the `useParams()` hook. This is a really simple and effective way to get those values from the routing context without drilling the props all the way down. 👏